website/docs/cli/commands/test.mdx

```diff
@@ -55,3 +55,80 @@ The `-cloud-run` option accepts a [private registry module source](/terraform/la
 You must provide a module from a _private_ registry, not the public Terraform registry.
 
 You must execute [`terraform login`](/terraform/cli/commands/login) before using this option, and ensure that your `hostname` argument matches the private registry hostname of your target module.
+
+## Example: Test Directory Structure and Commands
+
+The following directory structure represents an example directory tree for a Terraform module with tests and a setup module.
+
+```
+project/
+|-- main.tf
+|-- outputs.tf
+|-- terraform.tf
+|-- variables.tf
+|-- tests/
+|   |-- validations.tftest.hcl
+|   |-- outputs.tftest.hcl
+|-- testing/
+    |-- setup/
+        |-- main.tf
+        |-- outputs.tf
+        |-- terraform.tf
+        |-- variables.tf
+```
+
+At the root directory of the project, we have some typical Terraform configuration files: `main.tf`, `outputs.tf`, `terraform.tf`, and `variables.tf`. We have placed our test files, `validations.tftest.hcl` and `outputs.tftest.hcl` within the default tests directory: `tests`.
+
+In addition, we have created a [setup module](/terraform/language/tests#modules) for our tests within the `testing` directory.
+
+In order to execute our tests we should run `terraform test` from the root configuration directory as if we were running `terraform plan` or `terraform apply`. Despite our actual test files being in the nested `tests` directory, we still operate Terraform from the main configuration.
+
+We could also execute specific test files using the `-filter` option.
+
+Linux, Mac OS, and UNIX:
+
+```shell
+terraform test -filter=tests/validations.tftest.hcl`
```

Comment:
I don't think it makes a difference to the rendered website, but the extra backtick at the end of this line appears to be breaking Github's syntax highlighting 😆

Reply:
Fixed! Was just a typo!